lib/charms/smtp_integrator/v0/smtp.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env python3
```

Comment:
I think the shebang line is not necessary for the library :)

Reply:
Indeed. Thx